src/events/2024-2025/demos/README.md

```diff
@@ -0,0 +1,27 @@
+# Demo & Pitch Nights
+
+All Tech Start UCalgary projects are showcased to the world
+at the end of each academic year.
+At this event, people from different industries and backgrounds
+come and give us feedback on the projects.
+
+This is also a great opportunity for you as a professional and us
+as a club to connect and network with different companies.
+
+It's important to remember that as an entrepreneurial club, you will
+also be pitching your ideas to potential investors, so your hard work
+on the business strategy will be on display here. Pitching your start-up
+in a convincing manner is a critical skill, so we also give you the opportunity to
+practice this in the demos leading up to the final showcase.
+
+<!-- This year's showcase is scheduled for **April 27th, 2024**.
+To prepare for this, we want each team to demo and pitch their project to
+the club before the showcase. This will help you get experience and valuable
+feedback from fellow project teams, execs, and our partners in industry.-->
+
+## Dates to Remember
+
+| Date             |           Event |           Audience | Location |
+| ---------------- | --------------: | -----------------: | -------: |
+| 21 November 2024 | Elevator Pitche |         Tech Start |   ENG 03 |
```

Comment:
```suggestion
| 21 November 2024 | Elevator Pitch |         Tech Start |   ENGG 03 |
```

Reply:
`ENG` is the right name for the building. Fixed the Pitche typo 

![image](https://github.com/user-attachments/assets/e262a6f5-079f-426e-991b-3fea98f85819)